helm/prometheus/locals.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  helm_stable_repository = "https://kubernetes-charts.storage.googleapis.com"
+
+  storage_class_defaults = {
+    "aws" = "gp2"
```

Comment:
@limed didn't you recently talk me into ssd by default? :D

Reply:
my fault here, I looked up the storage classes and saw gp2 is now SSD

Reply:
For aws `gp2` is ssd so we just need to configure gcp for this.

For GCP I'm configuring the "faster" storage class (which is SSD) from the GKE module instead since this module is a generic module and by default standing up a GKE cluster doesn't setup an ssd storage class I don't want to make that assumption in the prometheus module that the faster storage class exists. So I'm going to configure prometheus to use the faster storage class in the GKE module, see 1e33a4a8b295f8fb7549e9fd3a53397250bf87cf